### PR TITLE
Gconinje guiderate

### PR DESCRIPTION
**Pull request by joakim-hove**
Internalize guiderate config items from GCONINJE

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/Group/Group.cpp [hunk 1]
```diff
@@ -539,6 +543,8 @@ Group::InjectionControls Group::injectionControls(Phase phase, const SummaryStat
     ic.target_void_fraction = UDA::eval_group_uda(inj.target_void_fraction, this->m_name, st, this->udq_undefined);
     ic.reinj_group = inj.reinj_group.value_or(this->m_name);
     ic.voidage_group = inj.voidage_group.value_or(this->m_name);
+    //ic.guide_rate = this->injection_properties.guide_rate;
+    //ic.guide_rate_def = this->injection_properties.guide_rate_def;
```
Comment (totto82) [inline src/opm/parser/eclipse/EclipseState/Schedule/Group/Group.cpp:548]: why are these commented out?

**Review by totto82 — APPROVED** [review]
Thanks. Looks good.

File: src/opm/parser/eclipse/EclipseState/Schedule/Group/Group.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/Group/Group.cpp:548]: Ehhhh - mistake ;-)

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**